Duplicati/Server/webroot/ngax/templates/commandline.html

```diff
@@ -14,22 +14,22 @@
 
             <div class="input textarea linklabel">
                 <label for="target"><a href class="target" ng-click="EditUriState = true">Target URL &gt;</a></label>
-                <textarea name="target" id="target" ng-model="TargetURL" placeholder="Enter a url, or click the 'Target URL &gt;' link"></textarea>
+                <textarea name="target" id="target" ng-model="TargetURL" placeholder="{{'Enter a url, or click the \'Target URL &gt;\' link | translate}}"></textarea>
```

Comment:
I am not perfectly sure if escaping would work this way... Fixing it if needed is really appreciated.

Reply:
Looks like this is what it looks like the other areas have done, so I think it is fine.